Common/src/main/resources/assets/hexcasting/lang/zh_cn.flatten.json5

```diff
@@ -997,636 +1233,795 @@
-      "pigments.colored.crafting.header": "色彩染色剂",
-      "pigments.colored.crafting.desc": "五彩缤纷的染色剂。",
-      "pigments.speical": "最后还有两种特殊的染色剂。$(item)灵魂闪光染色剂/$的颜色为我所独有，而$(item)空无染色剂/$则会重置为初始状态的紫橙混色。$(br2)$(italic)以及许多还没被研究出来的内在颜色。/$",
+      // The Work
 
+      the_work: {
+        "1": "我所见甚多。不可言之物。不可数之物。仅三词就能将我的意识从里翻到外然后把我的大脑抹在头颅黑暗的内壁然后腐朽为混沌和虚无。",
+        "2": "我已见断续的图案与酸蚀的设计图刻入我的眼睑。它们阴燃，它们舞蹈，它们嘲弄，它们$(italic)隐隐作痛/$。我突然有种强烈的$(italic)冲动/$，去绘制它们，创造它们，写就它们。将它们从我脆弱意识的黏糊桎梏中释放，让它们在世上再度闪耀荣光。$(p)众将见。$(p)众必将见。",
+      },
 
-      "edified.1": "用$(l:patterns/spells/blockworks#hexcasting:edify)$(action)启迪树苗/$就能将部分$(media)媒质/$注入树苗，就能造出$(l:items/edified)$(thing)启迪树/$。它们又高又尖，树皮上有特殊纹理。树叶则有三种不同颜色。",
-      "edified.2": "我觉得这种木材与$(hex)咒法学/$有着紧密的联系。但就算真有，我也完全找不到这联系在哪。它和普通木头如出一辙，不同只在于颜色不一样。$(br2)就现在而言，它们是很好的装饰用方块。$(br2)当然，也可用斧来去皮。",
-      "edified.crafting.desc": "$(italic)它们那周身洁白的巨大而光滑的树干，仅凭自己支撑起了茂密的树冠，投下绿荫与静谧。/$",
+      brainsweeping: {
+        "1": "我解开了一个秘密。我得知了真相。我也无法忘记真相带来的恐惧。这种想法掠过我的脑海。$(br2)我曾相信——我曾愚蠢地$(italic)相信/$——$(media)媒质/$只是思维遗留下的剩余能量。但我现在$(italic)知道/$它究竟是什么了：它是思维$(italic)含有的/$能量。",
+        "2": "媒质是在具有思维能力的生物思考时产生的，它的存在也使这些生物能够思考，就像是回环的绳结。我先前天真地拟人化为“自然”的实体实际上就是一个卓伟的媒质回环，又或许是所有回环的结合体，或者是……我思考它就会痛我有很多突触它们都能思考同时痛它们都能看见它们都能看见$(br2)我撑不住了。记笔记。赶快。",
+        "3": "这个世界中的村民还留有能被提取的感知力。将这种感知力放在方块中，扭曲它，改变它。由不同的思维模式造就的精密图案，关于他们工作和生活的抽象神经通路，都被投射到无生气的固态物质的原子中。$(br2)这就是$(l:patterns/great_spells/brainsweep)$(action)剥离意识/$的用途，那个提取过程。需将村民实体和目标方块放在栈中。扭曲意识则要消耗 10 个$(l:items/amethyst)$(item)充能紫水晶/$。",
+        budding_amethyst: "如下是一种实际用途。这个剥离过程接受任何职业的村民，前提是他们足够熟练。其他配方则有更具体的要求。我再也不必为$(media)媒质/$亲身沉入地狱了。",
+      },
 
+      spellcircles: {
+        "1": "我终于知道$(l:items/slate)$(item)石板/$的用途了。早已遗落的卓伟结构。镌刻其上的图案会按顺序自动运行。思维和能量在其中穿梭，一个接一个接一个接一个接一个穿梭穿梭穿梭——我不能我不能我不能跟着它的思维应该去理解。",
+        "2": "我需要$(l:greatwork/impetus)$(item)促动石/$作为一种自我维持的$(media)媒质/$波源来启动仪式。而产生的波就会随着$(l:items/slate)$(item)石板/$或是其他和这种波亲和性好的方块穿梭，一个接一个地收集碰到的图案。当波最终回到$(l:greatwork/impetus)$(item)促动石/$处时，所有碰到的图案就会按顺序运行。$(br2)$(media)媒质/$流出任何方块的方向必须唯一而确定，否则法术环会在分叉处失效。",
+        "3": "因此，法术“环”必须是一个封闭的图形，凹多边形和凸多边形均可，方向也随意。实际上，借助某些其他方块，法术环可在所有三个维度上随意排布。这种性质可能没什么大用，但我必须给大脑一点刺激，好让研究继续下去。",
+        "4": "真是精妙。法术环不会从我的物品栏或我的意识中汲取$(media)媒质/$，而是要通过漏斗（或类似装置）向$(l:greatwork/impetus)$(item)促动石/$提供晶态的$(media)媒质/$。$(br2)$(l:items/lens)$(item)探知透镜/$能以紫水晶粉为单位显示$(l:greatwork/impetus)$(item)促动石/$中的$(media)媒质/$量。",
+        "5": "然而，用法术环施放的法术有一个限制：它无法影响环外的事物。也即，它无法影响到能将整个法术环包起的最小长方体外的事物（有凹进部分的法术环仍能影响凹进部分内的事物）。",
+        "6": "在媒质波减弱前它只能通过有限个方块，但这个值很大，实际不大可能超过这个值。$(br2)相反地，有部分操作只能在法术环上执行，不过它们没有一个是法术。它们似乎都是处理法术环本身性质的操作。相关笔记$(l:patterns/circle)见此/$。",
+        "7": "我还在笔记中找到了一份草稿，是一个古代研究者使用的法术环。另一页上就是它的重绘图（挺抽象的）。$(br2)其中的图案会按逆时针运行，以$(l:patterns/basics#hexcasting:get_caster)$(action)意识之精思/$开始并以$(l:patterns/great_spells/teleport#hexcasting:teleport)$(action)卓越传送/$结束。",

```

Comment:
Link should end with `#hexcasting:teleport/great` instead of `#hexcasting:teleport`

Reply:
Fixed